sdk/eventgrid/eventgrid-namespaces/sample.env

```diff
@@ -0,0 +1,7 @@
+EVENT_GRID_NAMESPACES_ENDPOINT=<eventgrid endpoint>
+EVENT_GRID_NAMESPACES_KEY=<eventgrid key>
+TEST_MODE=<test mode>
```

Comment:
TEST_MODE should be removed as it is not used for samples

Reply:
Done (in next commit)